v1/src/main/java/com/google/cloud/teleport/templates/DLPTextToBigQueryStreaming.java

```diff
@@ -349,6 +354,7 @@ public interface TokenizePipelineOptions extends DataflowPipelineOptions {
 
     @TemplateParameter.ProjectId(
         order = 6,
+        groupName = "Source",
```

Comment:
Same. Move to "DLP Configuration" group.

Reply:
Done